services/net/contentmigration/TNO.Services.ContentMigration.csproj

```diff
@@ -11,14 +11,15 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\libs\net\services\TNO.Services.csproj" />
+    <!-- <ProjectReference Include="..\..\..\libs\net\services\TNO.Services.csproj" /> -->
```

Comment:
why comment this out?

Reply:
I left it in because if you want to test updates before building a new package it makes it easy to switch back to the project.  But if it causes confusion I can remove.